stl/inc/ostream

```diff
@@ -1038,10 +1038,14 @@ basic_ostream<_Elem, _Traits>& noemit_on_flush(basic_ostream<_Elem, _Traits>& _O
 
 template <class _Elem, class _Traits>
 basic_ostream<_Elem, _Traits>& flush_emit(basic_ostream<_Elem, _Traits>& _Ostr) {
+    const typename basic_ostream<_Elem, _Traits>::sentry _Ok(_Ostr);

```

Comment:
The wording in [[ostream.manip]/12](http://eel.is/c++draft/input.output#ostream.manip-12) is "Calls `os.flush()`. Then, if `os.rdbuf()` is a `basic_­syncbuf` [...], behaves as an unformatted output function". So I think the `sentry` should be inside the `if (_Sync_buf_ptr)` below.